plugins/woocommerce-admin/client/payments-welcome/style.scss

```diff
@@ -165,6 +178,72 @@
 				border-radius: 3px;
 			}
 		}
+
+		.connect-account__apms {
+			.components-card__header {
+				padding: $gap-large 10px $gap $gap-large;
+				h1 {
+					font-size: 20px;
+					line-height: 28px;
+				}
+			}
+			.components-card__body {
+				padding: 0;
+
+				.woocommerce-list__item {
+					> .woocommerce-list__item-inner {
+						padding: $gap-large;
+						.woocommerce-list__item-title {
+							color: $gray-900;
+							font-weight: 700;
+							font-size: 14px;
+							line-height: 18px;
+						}
+						.woocommerce-list__item-content {
+							color: $gray-700;
+							font-size: 13px;
+							line-height: 16px;
+							a {
+								text-decoration: none;
```

Comment:
_Nitpick_: As I mentioned before, I think it will be better to remove those `text-decoration: none` to be consistent with every other WC page. _There is also another one below_

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/34581/commits/579b1d63f1d8fdb76ca4449af6d23d873e438f09 and https://github.com/woocommerce/woocommerce/pull/34581/commits/62467e91647ad2907549fa097b24f50aa0da109d